public/app/core/logs_model.ts

```diff
@@ -284,6 +286,7 @@ export function logSeriesToLogsModel(logSeries: DataFrame[]): LogsModel | undefi
     return {
       series,
       timeField: fieldCache.getFirstFieldOfType(FieldType.time),
+      timeNanosecondField: isLoki && fieldCache.hasFieldNamed('tsNs') ? fieldCache.getFieldByName('tsNs') : undefined,
```

Comment:
Why does isLoki have to be here? Wouldn't the result be same without it?

Reply:
Aww, right. Forgot to remove this one. Thanks.